Makefile

```diff
@@ -59,7 +59,7 @@ endif
 
 
 ifeq ($(BUILD_OMS),BUILD_OMS)
-dsckit098: nx nxOMSAgent nxOMSPlugin nxOMSCustomLog nxFileInventory nxOMSGenerateInventoryMof
+dsckit098: nx nxOMSAgent nxOMSPlugin nxOMSCustomLog nxFileInventory nxOMSGenerateInventoryMof nxOMSAutomationWorker
```

Comment:
@D1v38om83r is correct. In the DIY case, we assumed customers will get the bits directly from the repo.